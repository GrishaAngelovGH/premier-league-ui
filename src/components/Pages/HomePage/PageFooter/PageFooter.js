import { Link } from 'react-router-dom'
import './PageFooter.scss'

export const ChevronDown = () => (
    <svg width='1em' height='1em' viewBox='0 0 16 16' className='bi bi-chevron-down' fill='white' xmlns='http://www.w3.org/2000/svg'>
        <path fillRule='evenodd' d='M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z' />
    </svg>
)

const PageFooter = () => (
    <div className='row'>
        <div className='col-md-12'>
            <div className='row no-gutters justify-content-between border p-5 text-secondary'>
                <div>All times are in Eastern European Time</div>
                <div>Feedback</div>
            </div>

            <Link to='/content' className='footer-btn'>
                <div className='row no-gutters justify-content-center border p-3'>
                    <div className='expand-btn mt-n4'>
                        <ChevronDown />
                    </div>
                </div>

                <div className='row no-gutters justify-content-center mt-n3'>
                    <span className='mt-n1'>See more</span>
                </div>
            </Link>
        </div>
    </div>
)

export default PageFooter