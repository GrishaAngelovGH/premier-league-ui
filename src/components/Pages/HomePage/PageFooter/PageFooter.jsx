import { Link } from 'react-router-dom'

export const ChevronDown = () => (
	<svg width='1em' height='1em' viewBox='0 0 16 16' className='bg-purple-900 text-white w-[30px] h-[30px] rounded-full p-2' fill=' white' xmlns='http://www.w3.org/2000/svg'>
		<path fillRule='evenodd' d='M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z' />
	</svg >
)

const PageFooter = () => (
	<div className="flex flex-col justify-center items-center">
		<div className='w-full text-center border border-t-0 border-gray-400 p-2 text-gray-500'>
			<div>All times are in Eastern European Time</div>
		</div>

		<Link to='/content' className="mt-2 flex flex-col items-center text-center text-decoration-none">
			<ChevronDown />
			<div>See more</div>
		</Link>
	</div>
)

export default PageFooter