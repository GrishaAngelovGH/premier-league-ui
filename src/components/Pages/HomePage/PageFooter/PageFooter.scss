.expand-btn {
    width: 30px;
    height: 30px;
    display: flex;
    cursor: pointer;
    justify-content: center;
    align-items: center;
    border-radius: 35px;
    background-color: #3f1052;
}

.footer-btn {
    color: #000000;
    &:hover {
        color: #000000;
        text-decoration: none;
    }
}
