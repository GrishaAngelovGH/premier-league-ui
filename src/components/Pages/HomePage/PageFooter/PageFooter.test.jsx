import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'

import PageFooter from './PageFooter'

test('should render PageFooter component', () => {
	const view = render(
		<MemoryRouter>
			<PageFooter />
		</MemoryRouter>
	)

	expect(view).toMatchSnapshot()
})