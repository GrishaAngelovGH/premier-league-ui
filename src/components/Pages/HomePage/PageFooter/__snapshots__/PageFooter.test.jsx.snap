// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render PageFooter component 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex flex-col justify-center items-center"
      >
        <div
          class="w-full text-center border p-2 text-gray-500"
        >
          <div>
            All times are in Eastern European Time
          </div>
        </div>
        <a
          class="mt-2 flex flex-col items-center text-center text-decoration-none"
          data-discover="true"
          href="/content"
        >
          <svg
            class="bg-purple-900 text-white w-[30px] h-[30px] rounded-full p-2"
            fill=" white"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z"
              fill-rule="evenodd"
            />
          </svg>
          <div>
            See more
          </div>
        </a>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex flex-col justify-center items-center"
    >
      <div
        class="w-full text-center border p-2 text-gray-500"
      >
        <div>
          All times are in Eastern European Time
        </div>
      </div>
      <a
        class="mt-2 flex flex-col items-center text-center text-decoration-none"
        data-discover="true"
        href="/content"
      >
        <svg
          class="bg-purple-900 text-white w-[30px] h-[30px] rounded-full p-2"
          fill=" white"
          height="1em"
          viewBox="0 0 16 16"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M1.646 4.646a.5.5 0 0 1 .708 0L8 10.293l5.646-5.647a.5.5 0 0 1 .708.708l-6 6a.5.5 0 0 1-.708 0l-6-6a.5.5 0 0 1 0-.708z"
            fill-rule="evenodd"
          />
        </svg>
        <div>
          See more
        </div>
      </a>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
